pwiz_tools/Skyline/SettingsUI/BuildLibraryDlg.cs

```diff
@@ -286,7 +297,7 @@ private bool ValidateBuilder(bool validateInputFiles)
                         // TODO: Probably need to validate that all the libraries can be loaded into memory with progress UI
                     }
 
-                    // TODO: Create CarafeLibraryBuilder class with everything necessary to build a library
+                    // TODO: Create AlphapeptdeepLibraryBuilder class with everything necessary to build a library

```

Comment:
Seems like the original text is still appropriate.

Reply:
Revert to original text.